src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AppointmentDtos.kt

```diff
@@ -156,8 +156,13 @@ data class VideoLinkAppointmentDto(
   val bookingId: Long,
   @ApiModelProperty(value = "Appointment id, maps to nomis event id", example = "1")
   val appointmentId: Long,
-  @ApiModelProperty(value = "The location of the court that requires the appointment", example = "York Crown Court")
+
```

Comment:
Jamie is making some changes on this DTO - https://github.com/ministryofjustice/whereabouts-api/pull/299/files. I assume one of you will have to do some merging.

Reply:
Sigh